extensions/amp-analytics/0.1/vendors.js

```diff
@@ -541,6 +544,73 @@ export const ANALYTICS_CONFIG = {
       'image': true,
     },
   },
+
+  'mpulse': {
+    'requests': {
+      'onvisible': 'https://${beacon_url}?' +
```

Comment:
@querymetrics @avimehta what would this var `beacon_url` resolve to? I'm seeing exception for `mpulse` in the `examples/analytics-vendors.amp.html`